index.html

```diff
@@ -1,12 +1,262 @@
 <!DOCTYPE html>
 <html lang="ru">
+
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Гостиницы Седоны</title>
+    <link rel="stylesheet" href="styles.css">
   </head>
+
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
 
+    <!-- Шапка -->
+
+
+    <header class="page-header" data-test="header">
+      <nav class="page-navigation">
+        <ul class="navigation-list">
+
+          <!-- Logo -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="index.html" aria-current="page">
+              <img class="page-header-logo" src="images/same/logo.svg" width="140" height="70"
+                alt="Логотип сайта города Седоны.">
+            </a>
+          </li>
+
+          <!-- Главная -->
+          <li class="navigation-item">
+            <a class="navigation-link navigation-link-current" href="page.html">Главная</a>
+          </li>
+
+          <!-- О Седоне -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="about_Sedona.html">О Седоне</a>
+          </li>
```

Comment:
```suggestion
          <li class="navigation-item">
            <a class="navigation-link" href="#">О Седоне</a>
          </li>
```
Не придумывай страниц. 
Если не знаем куда ведет ссылка, просто ставим `#`. 
Представь, что ты взяла верстку и тебе нужно из неё сделать реальный сайт. И все ссылки заменить на настоящие. Можно быстренько по поиску найти все решетки, а можно мучительно глазами искать и читать все ссылки и проверять они ведут уже куда надо или они неверные.

Reply:
заменила на #